compiler-rt/lib/sanitizer_common/sanitizer_linux_libcdep.cpp

```diff
@@ -29,7 +29,13 @@
 #  include "sanitizer_solaris.h"
 
 #  if SANITIZER_NETBSD
-#    define _RTLD_SOURCE  // for __lwp_gettcb_fast() / __lwp_getprivate_fast()
+#    // for __lwp_gettcb_fast() / __lwp_getprivate_fast()
+#    include <sys/param.h>
```

Comment:
Can this be moved under the if condition, since it didn't seem to be needed for older NetBSD versions?

Reply:
Ah, I see, thanks!